newbank/test/ServerTestScenarios.java

```diff
@@ -0,0 +1,29 @@
+package newbank.test;
+
+import newbank.server.NewBank;
+
+// How to implement test:
+// 1. Define a class
+// 2. Define test methods where:
+//    accessibility is public
+//    no parameters
+//    return value type is void
+// 3. register the test class the runInTestMode method.
+public class ServerTestScenarios {
+
+  public void ShowMyAccountsReturnsListOfAllCustomersAccountsAlongWithCurrentBalance() {
+
+    String bhagy =
+        NewBank.getBank()
+            .processRequest(NewBank.getBank().checkLogInDetails("Bhagy", null), "SHOWMYACCOUNTS");
+
+    newbank.test.NBUnit.AssertEqual("Main: 1000.0" + System.lineSeparator(), bhagy);
```

Comment:
Hi Shinichi sorry one last thing then it's ready to merge... to make your test compatible with the password feature recently added can you change the test login credentials to checkLogInDetails("Bhagy", **"1"**) and checkLogInDetails("Christina", **"2"**) please?